<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartupDialogClass</class>
 <widget class="QDialog" name="StartupDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCommandLinkButton" name="newProjectCmdButton">
     <property name="text">
      <string>New project (empty)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCommandLinkButton" name="imageProjectCmdButton">
     <property name="text">
      <string>New from an image...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCommandLinkButton" name="openProjectCmdButton">
     <property name="text">
      <string>Open a project...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QLabel" name="label">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
